src/TensorOneD.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+
+#include "config.h"
+
+#include "TensorOneD.hpp"
+
+#include <math.h>
```

Comment:
In the C programming language, we add `.h` to the end of the name of standard headers.
In the C++ programming language, we add `c` to the front of the name of standard headers.
```cpp
#include <cmath>
```